### PR TITLE
docs: adding job-queue docs

### PR DESCRIPTION
**Pull request by eecsliu**
## Description
Adds new job queue docs and connects them to the existing scheduler docs.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/features/scheduling.txt [hunk 1]
```diff
@@ -109,6 +109,9 @@ Here is an example of how the priority scheduler behaves with preemption enabled
    priority 1 distributed training experiment starts running. Once that experiment is complete,
    distributed training experiment with priority 2 restarts.
 
+The priority scheduler can be used with Determined's :ref:`job-queue`, which allows more insight and
+control over scheduling decisions.
```
Comment (hamidzr) [inline docs/features/scheduling.txt:113]: do we leave "control" out for now?

File: docs/features/job-queue.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+To view the jobs that are currently active in the cluster, use the following command in the CLI:
+
+.. code:: bash
+
+   det job list
+
+This will display a table of results, ordered by scheduling order. Note that the scheduling order is
+not necessarily the priority of the job. In addition to the order of the job, users can see the
+state of the jobs and how many slots each job is using. Jobs can be in the following states: queued,
+running, backfilled, or preempted. Jobs that have completed or errored out are not counted as active
```
Comment (hamidzr) [inline docs/features/job-queue.txt:22]: > obs can be in the following states: queued, running, backfilled, or preempted.

IMO `preempted` used here could be confusing.

wdy think about mapping these to the states that users would see in the api response or the UI? 

in those places, we report one of 3 queued, scheduled, or scheduled backfilled.

**Review by hamidzr — APPROVED** [review]

File: docs/features/scheduling.txt (hunk 1 above)
Comment (hamidzr) [reply docs/features/scheduling.txt:113]: same for the k8 version of this

File: docs/features/job-queue.txt (hunk 2 above)
Comment (eecsliu) [reply docs/features/job-queue.txt:22]: ahhh good catch. This was just me spilling my thoughts and forgot to go back to edit/verify.

File: docs/features/scheduling.txt (hunk 1 above)
Comment (eecsliu) [reply docs/features/scheduling.txt:113]: will do

**Comment by hamidzr** [comment]
two other points worth mentioning I think would be 
- the job queue can be viewed on the webui
- not all schedulers provide job ordering information
- controlling and adjusting the queue would become available in an upcoming release..

**Pull request merged**